source/about_jena/security-advisories.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Jena Security Advisories
+---
+
+# Security Issue Policy
+
+## Process
```

Comment:
The title becomes an H1 on the page. So that's two H1 then an H2.

---
![image](https://user-images.githubusercontent.com/101362/202807156-428892d1-c5a3-4929-95de-f38a6b3a7ed4.png)

---

For appearance, drop the "# " ; adds some text to separate the H1 from the H2 Process.

Reply:
Have added a brief into paragraph